<ui version="4.0" >
 <class>ConfigureGlLights</class>
 <widget class="QDialog" name="ConfigureGlLights" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Configure GL Lights</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <widget class="Q3GroupBox" name="groupBox1" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>87</y>
     <width>271</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title" >
    <string/>
   </property>
   <widget class="QPushButton" name="ambient_color" >
    <property name="geometry" >
     <rect>
      <x>140</x>
      <y>90</y>
      <width>79</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>Ambient Color</string>
    </property>
    <property name="flat" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ambient_color_label" >
    <property name="geometry" >
     <rect>
      <x>225</x>
      <y>90</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize" >
     <size>
      <width>25</width>
      <height>0</height>
     </size>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="scaledContents" >
     <bool>true</bool>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="apply_button" >
    <property name="geometry" >
     <rect>
      <x>193</x>
      <y>152</y>
      <width>77</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel3_2" >
    <property name="geometry" >
     <rect>
      <x>18</x>
      <y>33</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>Position Y</string>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="posx" >
    <property name="geometry" >
     <rect>
      <x>70</x>
      <y>5</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize" >
     <size>
      <width>120</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel3" >
    <property name="geometry" >
     <rect>
      <x>18</x>
      <y>5</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>Position X</string>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="posy" >
    <property name="geometry" >
     <rect>
      <x>70</x>
      <y>33</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize" >
     <size>
      <width>120</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel3_3" >
    <property name="geometry" >
     <rect>
      <x>18</x>
      <y>61</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>Position Z</string>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="diffuse_color_label" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>86</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize" >
     <size>
      <width>25</width>
      <height>0</height>
     </size>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="scaledContents" >
     <bool>true</bool>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="diffuse_color" >
    <property name="geometry" >
     <rect>
      <x>17</x>
      <y>86</y>
      <width>77</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>Diffuse Color</string>
    </property>
    <property name="flat" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="specular_color_label" >
    <property name="geometry" >
     <rect>
      <x>102</x>
      <y>119</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize" >
     <size>
      <width>25</width>
      <height>0</height>
     </size>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="scaledContents" >
     <bool>true</bool>
    </property>
    <property name="wordWrap" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="specular_color" >
    <property name="geometry" >
     <rect>
      <x>17</x>
      <y>119</y>
      <width>79</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>Specular Color</string>
    </property>
    <property name="flat" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="posz" >
    <property name="geometry" >
     <rect>
      <x>70</x>
      <y>61</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize" >
     <size>
      <width>120</width>
      <height>0</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="Q3Frame" name="frame3" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>274</y>
     <width>272</width>
     <height>133</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="Q3ListBox" name="saved_configs" >
    <property name="geometry" >
     <rect>
      <x>119</x>
      <y>9</y>
      <width>144</width>
      <height>115</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="load_config" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>102</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>Load Configuration</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_config" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>41</y>
      <width>102</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>Save Configuration</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="light_enabled" >
   <property name="geometry" >
    <rect>
     <x>145</x>
     <y>17</y>
     <width>62</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Enabled</string>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>64</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text" >
    <string>Selected light</string>
   </property>
   <property name="wordWrap" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="selected_light" >
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>16</y>
     <width>59</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text" >
     <string>Light 0</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>Light 1</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>Light 2</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>Light 3</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>Light 4</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>Light 5</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>Light 6</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>Light 7</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="textLabel2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>50</y>
     <width>48</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text" >
    <string>Light type</string>
   </property>
   <property name="wordWrap" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="type_point" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>64</x>
     <y>57</y>
     <width>48</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Point</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="type_parallel" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>118</x>
     <y>57</y>
     <width>58</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Parallel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_button" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>417</y>
     <width>132</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ListBox</class>
   <extends>Q3Frame</extends>
   <header>q3listbox.h</header>
  </customwidget>
 </customwidgets>
 <includes>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>selected_light</sender>
   <signal>activated(int)</signal>
   <receiver>ConfigureGlLights</receiver>
   <slot>OnSelectLight(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ok_button</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureGlLights</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>diffuse_color</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureGlLights</receiver>
   <slot>OnSelectDiffuse()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>specular_color</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureGlLights</receiver>
   <slot>OnSelectSpecular()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>apply_button</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureGlLights</receiver>
   <slot>OnApplyLightSettings()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>load_config</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureGlLights</receiver>
   <slot>LoadConfiguration()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save_config</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureGlLights</receiver>
   <slot>SaveConfiguration()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saved_configs</sender>
   <signal>selected(int)</signal>
   <receiver>ConfigureGlLights</receiver>
   <slot>SaveConfiguration(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
